.travis.yml

```diff
@@ -9,10 +10,11 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-5
+    - clang
```

Comment:
Both compilers are used. For each target`scan-build` will build the object file as normal with`g++-5` then statically-analyze the target with `clang++`. `clang++` in analysis mode doesn't create any object files.